src/core/net/ip6_mpl.hpp

```diff
@@ -203,61 +388,106 @@ class Mpl
      *
      * @param[in]   aMessage  A reference to the message.
      * @param[in]   aAddress  A reference to the IPv6 Source Address.
-     * @param[out]  aForward  TRUE if this message should be forwarded, FALSE otherwise.
      *
      * @retval kThreadError_None  Successfully processed the MPL option.
      * @retval kThreadError_Drop  The MPL message is a duplicate and should be dropped.
      *
      */
-    ThreadError ProcessOption(const Message &aMessage, const Address &aAddress, bool &aForward);
+    ThreadError ProcessOption(Message &aMessage, const Address &aAddress);
+
+    /**
+     * This method introduce MPL packet into MPL Domain. When MPL module acts as MPL Forwarder
```

Comment:
nit: `introduce MPL` -> `introduces an MPL`
nit: `When MPL module acts as MPL Forwarder` -> `When the MPL module acts as an MPL Forwarder`